frontend/src/components/GetActionItems.js

```diff
@@ -0,0 +1,105 @@
+import React, { useState, useEffect } from "react";
+import { useSelector } from "react-redux";
+
+import { ACTION_PLAN_URL, DELETE_ACTION } from "../utils/constants";
+
+import styled from "styled-components";
+import swal from "sweetalert";
+import { CloseIcon } from "@chakra-ui/icons";
+
+const GetActionItems = () => {
+  const [actionItems, setActionItems] = useState([]);
+
+  const retroId = useSelector((store) => store.retro._id);
+
+  useEffect(() => {
+    fetch(ACTION_PLAN_URL(`${retroId}`))
+      .then((res) => res.json())
+      .then((data) => setActionItems(data));
+  }, [retroId]);
+
+  const RefreshButton = () => {
+    window.location.reload("Refresh");
+  };
+
+  const onDeleteClick = (id) => {
+    swal({
+      title: "Are you sure?",
+      text: "Once deleted, you won't be able to recover this action",
+      icon: "warning",
+      buttons: true,
+      dangerMode: true,
+    }).then((willDelete) => {
+      if (willDelete) {
+        const options = {
+          method: "DELETE",
+        };
+        fetch(DELETE_ACTION(`${id}`), options);
+        swal("Your action has been deleted", {
+          icon: "success",
+        }).then(RefreshButton);
```

Comment:
It's a bit too risky to send fetch request without checking if it was successfull in .then() block - instead I would suggest you to write it this way :

```
fetch(DELETE_ACTION(`${id}`), options)
  .then(res => res.json())
  .then(data => {
    if (data.success) {
      swal("Your action has been deleted", {
        icon: "success",
      }).then(RefreshButton);
    }
  })
```